packages/css/types/index.d.ts

```diff
@@ -14,6 +18,6 @@ export { Interpolation, SerializedStyles }
 
 export default function css(
   template: TemplateStringsArray,
-  ...args: Array<Interpolation>
+  ...args: Array<CSSInterpolation>
```

Comment:
@Andarist this change fixed the test where `css` was too permissive. Let me know if you forsee any issues

Reply:
Should be fine 👍